### PR TITLE
Fix ScheduleNewsletter acceptance test [MAILPOET-4221]

### PR DESCRIPTION
**Pull request by costasovo**
It is not an elegant fix but works.
[MAILPOET-4221]

[MAILPOET-4221]: https://mailpoet.atlassian.net/browse/MAILPOET-4221?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/tests/acceptance/Newsletters/ScheduleNewsletterCest.php [hunk 1]
```diff
@@ -51,8 +52,17 @@ public function scheduleStandardNewsletterButtonCaption(\AcceptanceTester $i) {
...
+    $i->wantTo('Pick today‘s date');
     $i->waitForElement('form input[name=date]');
     $i->click('form input[name=date]');
+    // The calendar preselects tomorrow's date, making today's date not clickable on the last day of a month.
+    // In case it is not clickable try switching to previous month
+    try {
+      $i->waitForElementClickable(['class' => 'react-datepicker__day--today'], 1);
+    } catch (TimeoutException $e) {
+      $i->click(['class' => 'react-datepicker__navigation--previous']);
+      $i->waitForElementClickable(['class' => 'react-datepicker__day--today'], 1);
```
Comment (samnajian) [inline mailpoet/tests/acceptance/Newsletters/ScheduleNewsletterCest.php:64]: Hard to reason, but the default timeout value would be less prone to timeout issues

Comment (samnajian) [inline mailpoet/tests/acceptance/Newsletters/ScheduleNewsletterCest.php:64]: Hard to reason, but the default timeout value would be less prone to timeout issues

**Comment by samnajian** [comment]
LGTM, please see the comment

**Comment by samnajian** [comment]
LGTM, please see the comment

**Review by samnajian — DISMISSED** [review]

**Comment by costasovo** [comment]
@samnajian I updated the second timeout. Could you please re-approve?

**Review by veljkho — APPROVED** [review]

**Pull request merged**